docs/modules/ROOT/pages/explanations/decisions/mongodb.adoc

```diff
@@ -0,0 +1,37 @@
+= MongoDB
+
+== Problem
+
+We want to Provide MongoDB via AppCat.
+But their licensing might be an issue.
+This decision looks closely at the licensing and alternative products that can be used as drop-in replacements for MongoDB.
+
+== Proposals
+
+Proposal 1: Use MongoDB Community Edition::
+MongoDB community edition is licensed under the https://en.wikipedia.org/wiki/Server_Side_Public_License[SSPL].
+The SSPL dictates that the source code for the tooling that provides the services, needs to be available to the public as well.
+However, it would also need to be licensed under the SSPL.
+
+Proposal 2: Use MongoDB Enterprise Operator::
+MongoDB Enterprise Operator can be used to manage MongoDB instances on Kubernetes.
+The licensing allows to use it as a SaaS according to their docs.
```

Comment:
Since the decision basically hinges on this statement, a link (and potentially a quote here) to the mentioned docs should be included.

Reply:
Hmmm... I can't actually find the mentioning of the SaaS again.

However, I since learned we've been in contact with MongoDB already and even have a concrete quote. I adjust the wording here.